sdg: quotients and fermat-hadamard rings

stollluk opened the pull request:
Some material on quotients by congruence relations over endomorphism theories and the relationship with ring-theoeretic ideals.

stollluk committed:
Add paragraph on direct products and quotients

stollluk committed:
Add subsection on 'ringed endomorphism theories', Fermat-Hadamard rings

stollluk committed:
Merge branch 'felixwellen:main' into main

stollluk committed:
Add small comment

felixwellen commented:
The statement alluded to in the comment above is prop. 1.9 in the sdg notes (state of this PR/commit).

markrd-williams commented:
The property you suggested, regularity of generators of free algebras seems reasonable to me.

The property of $X-Y$ being regular in $F(n+2)$ reduces to $X$ being regular in $F(n+2)$, by an automorphism swapping $X$ and $X-Y$. Further it is enough to ask for $X$ being regular in $F(1)$ to get this property.

This reads as some sort of continuity condition to me, if the base ring is a field, then $X$ regular says: for all $f : R \to R$ s.t $f(r) = 0$ for all $r \neq 0$ then $f = 0$.

stollluk committed:
sdg: add details to proof that R is field, remark on nilpotent elements of R

stollluk committed:
sdg: Fix proof

Every algebraic theory T can be recovered as
endomorphism theory of a generic T-algebra in an
appropriate Grothendieck topos with a
*subcanonical* topology.
